packages/api-clients/open-api/delegationApi.yml

```diff
@@ -253,6 +253,49 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Problem"
+  /consumer/delegations/{delegationId}/approve:
+    parameters:
+      - $ref: "#/components/parameters/CorrelationIdHeader"
+      - name: delegationId
+        in: path
+        description: The delegation id
+        required: true
+        schema:
+          type: string
+          format: uuid
+    post:
+      description: Approves a consumer delegation
+      summary: Consumer delegation approval
+      tags:
+        - consumer
```

Comment:
I think we need to add the tag at the beginning of the file

Reply:
Whoops you're right, added it in a further PR: https://github.com/pagopa/interop-be-monorepo/pull/1220/commits/c753e07c050e0b12bf034f5b8dd213f371eb8ef6